### DISCUSSION TITLE
Log folders appear in many places

### DISCUSSION THREAD

**Discussion by chenqinggang001** [seleniumbase/SeleniumBase#1413] [Q&A] [answered]
The path of each test case varies and the location of the log folder varies. Can specify a path？

![image](https://user-images.githubusercontent.com/74290987/178922158-f9175c2a-a87e-4335-b665-2a5ee478ea64.png)

**Comment by mdmintz** [comment, marked as answer]
The ``latest_logs/`` folder is created from the location where ``pytest`` is called. Not configurable, for reasons outlined in https://github.com/seleniumbase/SeleniumBase/issues/486
